### DISCUSSION TITLE
Seeing possible related list without actually adding them

### DISCUSSION THREAD

**Discussion by isrbaral** [arnoudkooi/SN-Utils#419] [Ideas]
Hey, 

I think it would be kinda cool if you there was a way to see related lists for a record, just like you can open the versions list for a specific record even if the versions RL hasns't been added to the view. This way, if you just want to quickly check something you don't need to make a configuration change or go hunting for the table and figure out the correct filtering, you can just slashcommand your way to whichever RL that record has available, even if it's not currently visible.

**Comment by arnoudkooi** [comment]
I have looked at a way to add "add hoc" related lists before, but couldn't figure out how. The way you describe it similar to the versions way, may be a nice alternative. I'll have a look at it when I find some time. I'll move this to a discussion for now.

**Reply by isrbaral** [reply]
Awesome thanks! I wish I could help with this
